### ISSUE TITLE
Update automatically doesn't work anymore

### ISSUE THREAD

**Issue by craeckor** [LinkStackOrg/LinkStack#681]
### LinkStack version

4.6.1 --> 4.7.0

### Description

Today, i tried to update my LinkStack but then i realized it didn't actually update, in fact, it did nothing. First, i thought that my Cache-Service i use isn't up-to-date but then i logged-in directly without any Cache-Service any the Instance is still on 4.6.1.
Then i tried it again, same thing happened. It said that it's done, no error at all, but again, NOTHING HAPPENED.
Then i thought that something doesn't work correctly with PHP, so i turned on debug-log and access-log and updated again. Same thing happened. It said it's done but it wasn't. And no logs at all.
access.log and debug.log are mentioned under additional context.
P.S:
It always worked with older Versions.

### Details about your system

OS: Debian GNU/Linux 12 (bookworm)
Virtualization: lxc
Kernel: Linux 6.5.11-7-pve (Proxmox-VE)
Webserver: OpenLiteSpeed V 1.7.18
PHP-Version (LSPHP): 8.2.11 (All components required by LinkStack are installed and up-to-date)

### How to reproduce

Login to Dashboard
Click on the Update-Icon
Click on update available
Click on Update automatically
Wait
Click on Admin Panel
Nothing happend, Update-Notification is still here and LinkStack is still on Version 4.6.1

### Possible Solution

Workaround: Manually update LinkStack.

### Additional Context

[access.log](https://github.com/LinkStackOrg/LinkStack/files/13817508/access.log)
[debug.log](https://github.com/LinkStackOrg/LinkStack/files/13817509/debug.log)

**Comment by JulianPrieber** [comment]
Did your file permissions change?
Could you also provide parts of your Laravel log?

**Comment by craeckor** [comment]
Found the Problem.
I added a single file to the root-Directory named sitemap.xml. This file had nothing to do with LinkStack and was a simple sitemap. This file had wrong permissions (-rw-r--r--  1 root   root) and (i believe, because after i moved the file one folder back, it worked) because of this single file, it somehow failed to update LinkStack. (Makes no sense me)
Here is my Laravel-Log: [laravel.log](https://github.com/LinkStackOrg/LinkStack/files/13820608/laravel.log)
